io/src/test/java/org/commonjava/maven/ext/io/XMLIOTest.java

```diff
@@ -212,6 +212,15 @@ public void modifyNotFoundFile ()
         assertEquals( 0, nodeList.getLength() );
     }
 
+    @Test
+    public void findArtemis() throws ManipulationException, XPathExpressionException
+    {
+        Document doc = xmlIO.parseXML( xmlFile );
+        XPath xPath = XPathFactory.newInstance().newXPath();
+        NodeList nodeList = (NodeList) xPath.evaluate( "//include[starts-with(.,\"org.apache.tomcat\")]", doc, XPathConstants.NODESET );
```

Comment:
I am not really sure I understand why the method is called `findArtemis` but the tests looks for `org.apache.tomcat`.

Reply:
Probably because its looking at an xml file called `activemq-artemis`. :-) I'll rename it